bpf/go_grpc.h

```diff
@@ -150,7 +150,7 @@ int uprobe_netFdReadGRPC(struct pt_regs *ctx) {
     bpf_dbg_printk("tr %llx", tr);
     if (tr) {
         go_addr_key_t tr_key = {};
-        go_addr_key_from_id(&tr_key, tr);
+        go_addr_key_from_id(&tr_key, (u64 *)tr);
```

Comment:
This looks strange, given `go_addr_key_from_id` takes a `void*`, I'd expect this to have no side-effect.